presto/presto.py

```diff
@@ -185,7 +187,8 @@ def __init__(
         norm_layer=nn.LayerNorm,
     ):
         super().__init__()
-        self.norm1 = norm_layer(dim)
+        self.norm1 = norm_layer(dim, device=device)
+        # self.norm1 = norm_layer(dim)
```

Comment:
Why are these `to_device` calls necessary? They should all be handled when you call `model.to(device)`